dhcpv4/nclient4/ipv4.go

```diff
@@ -26,16 +27,17 @@ import (
 )
 
 const (
-	versIHL     = 0
-	tos         = 1
-	totalLen    = 2
-	id          = 4
-	flagsFO     = 6
-	ttl         = 8
-	protocol    = 9
-	checksumOff = 10
-	srcAddr     = 12
-	dstAddr     = 16
+	versIHL        = 0
+	tos            = 1
+	totalLen       = 2
+	id             = 4
+	flagsFO        = 6
+	ttl            = 8
+	protocol       = 9
+	checksumOff    = 10
+	srcAddr        = 12
+	dstAddr        = 16
+	ipVersionShift = 4
```

Comment:
Maybe move this to a separate group below `ipVersion` for consistency with the other file you are copying it from